xaitk_saliency/impls/gen_detector_prop_sal/drise_scoring.py

```diff
@@ -124,7 +124,9 @@ def generate(
         s = s.max(axis=1)
 
         # Weighting perturbed regions by similarity
-        sal = weight_regions_by_scalar(s, perturbed_masks)
+        sal = weight_regions_by_scalar(s,
+                                       perturbed_masks,
+                                       inv_masks=False)
```

Comment:
out of curiousity, is there a reason that inv_masks is "True" by default?

Reply:
Probably a historical choice, the first algorithm to use that was the `OcclusionScoring` implementation (with sliding windows), which required a mask inversion when computing saliency. We could discuss changing the default behavior now that this function is used in multiple places.